jsonrpc/filter_manager.go

```diff
@@ -293,8 +297,11 @@ func (f *FilterManager) Run() {
 
 		case <-timeoutCh:
 			// timeout for filter
-			if !f.Uninstall(filterBase.id) {
-				f.logger.Error("failed to uninstall filter", "id", filterBase.id)
+			// if filter still exists
+			if filterBase != nil {
+				if !f.Uninstall(filterBase.id) {
+					f.logger.Error("failed to uninstall filter", "id", filterBase.id)
+				}
```

Comment:
Please unwrap this spaghetti

Reply:
Fixed 81645b5447a911cbb2a8d1ade24eb4b4e16a7a55